src/lib/fbxAnimation/mixamoVRMRigMap.ts

```
@@ -0,0 +1,103 @@
+	"mixamorigRightHandIndex3" |
+	"mixamorigRightHandThumb1" |
+	"mixamorigRightHandThumb2" |
+	"mixamorigRightHandThumb3" |
+	"mixamorigLeftUpLeg" |
+	"mixamorigLeftLeg" |
+	"mixamorigLeftFoot" |
+	"mixamorigLeftToeBase" |
+	"mixamorigRightUpLeg" |
+	"mixamorigRightLeg" |
+	"mixamorigRightFoot" |
+	"mixamorigRightToeBase";
```

ESLint

[error] Replace `·⏎↹"mixamorigHips"·|·"mixamorigSpine"·|·"mixamorigSpine1"·|·"mixamorigSpine2"·|⏎↹"mixamorigNeck"·|·"mixamorigHead"·|·"mixamorigLeftShoulder"·|·"mixamorigLeftArm"·|·⏎↹"mixamorigLeftForeArm"·|·"mixamorigLeftHand"·|·"mixamorigLeftHandThumb1"·|·⏎↹"mixamorigLeftHandThumb2"·|·"mixamorigLeftHandThumb3"·|⏎↹"mixamorigLeftHandIndex1"·|⏎↹"mixamorigLeftHandIndex2"·|·⏎↹"mixamorigLeftHandIndex3"·|⏎↹"mixamorigLeftHandMiddle1"·|·⏎↹"mixamorigLeftHandMiddle2"·|⏎↹"mixamorigLeftHandMiddle3"·|⏎↹"mixamorigLeftHandRing1"·|⏎↹"mixamorigLeftHandRing2"·|⏎↹"mixamorigLeftHandRing3"·|⏎↹"mixamorigLeftHandPinky1"·|⏎↹"mixamorigLeftHandPinky2"·|⏎↹"mixamorigLeftHandPinky3"·|⏎↹"mixamorigRightShoulder"·|⏎↹"mixamorigRightArm"·|⏎↹"mixamorigRightForeArm"·|⏎↹"mixamorigRightHand"·|⏎↹"mixamorigRightHandPinky1"·|⏎↹"mixamorigRightHandPinky2"·|⏎↹"mixamorigRightHandPinky3"·|⏎↹"mixamorigRightHandRing1"·|⏎↹"mixamorigRightHandRing2"·|⏎↹"mixamorigRightHandRing3"·|⏎↹"mixamorigRightHandMiddle1"·|⏎↹"mixamorigRightHandMiddle2"·|⏎↹"mixamorigRightHandMiddle3"·|⏎↹"mixamorigRightHandIndex1"·|⏎↹"mixamorigRightHandIndex2"·|⏎↹"mixamorigRightHandIndex3"·|⏎↹"mixamorigRightHandThumb1"·|⏎↹"mixamorigRightHandThumb2"·|⏎↹"mixamorigRightHandThumb3"·|⏎↹"mixamorigLeftUpLeg"·|⏎↹"mixamorigLeftLeg"·|⏎↹"mixamorigLeftFoot"·|⏎↹"mixamorigLeftToeBase"·|⏎↹"mixamorigRightUpLeg"·|⏎↹"mixamorigRightLeg"·|⏎↹"mixamorigRightFoot"·|⏎↹"mixamorigRightToeBase";` with `⏎··|·'mixamorigHips'⏎··|·'mixamorigSpine'⏎··|·'mixamorigSpine1'⏎··|·'mixamorigSpine2'⏎··|·'mixamorigNeck'⏎··|·'mixamorigHead'⏎··|·'mixamorigLeftShoulder'⏎··|·'mixamorigLeftArm'⏎··|·'mixamorigLeftForeArm'⏎··|·'mixamorigLeftHand'⏎··|·'mixamorigLeftHandThumb1'⏎··|·'mixamorigLeftHandThumb2'⏎··|·'mixamorigLeftHandThumb3'⏎··|·'mixamorigLeftHandIndex1'⏎··|·'mixamorigLeftHandIndex2'⏎··|·'mixamorigLeftHandIndex3'⏎··|·'mixamorigLeftHandMiddle1'⏎··|·'mixamorigLeftHandMiddle2'⏎··|·'mixamorigLeftHandMiddle3'⏎··|·'mixamorigLeftHandRing1'⏎··|·'mixamorigLeftHandRing2'⏎··|·'mixamorigLeftHandRing3'⏎··|·'mixamorigLeftHandPinky1'⏎··|·'mixamorigLeftHandPinky2'⏎··|·'mixamorigLeftHandPinky3'⏎··|·'mixamorigRightShoulder'⏎··|·'mixamorigRightArm'⏎··|·'mixamorigRightForeArm'⏎··|·'mixamorigRightHand'⏎··|·'mixamorigRightHandPinky1'⏎··|·'mixamorigRightHandPinky2'⏎··|·'mixamorigRightHandPinky3'⏎··|·'mixamorigRightHandRing1'⏎··|·'mixamorigRightHandRing2'⏎··|·'mixamorigRightHandRing3'⏎··|·'mixamorigRightHandMiddle1'⏎··|·'mixamorigRightHandMiddle2'⏎··|·'mixamorigRightHandMiddle3'⏎··|·'mixamorigRightHandIndex1'⏎··|·'mixamorigRightHandIndex2'⏎··|·'mixamorigRightHandIndex3'⏎··|·'mixamorigRightHandThumb1'⏎··|·'mixamorigRightHandThumb2'⏎··|·'mixamorigRightHandThumb3'⏎··|·'mixamorigLeftUpLeg'⏎··|·'mixamorigLeftLeg'⏎··|·'mixamorigLeftFoot'⏎··|·'mixamorigLeftToeBase'⏎··|·'mixamorigRightUpLeg'⏎··|·'mixamorigRightLeg'⏎··|·'mixamorigRightFoot'⏎··|·'mixamorigRightToeBase'`

(prettier/prettier)